client/client.go

```
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+)
+
+func main() {
+	clt, _ := net.Dial("tcp", "127.0.0.1:8083")
+	defer clt.Close()
+	for {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Print("Message to server: ")
+		message, _ := reader.ReadString('\n')
```

The same: do not ignore an error.